documentation/staging/content/_index.md

```diff
@@ -1,22 +1,28 @@
 ### Oracle WebLogic Server Kubernetes Operator
 
-Oracle is finding ways for organizations using WebLogic Server to run important workloads, to move those workloads into the cloud. By certifying on industry standards, such as Docker and Kubernetes, WebLogic now runs in a cloud neutral infrastructure. In addition, we've provided an open source Oracle WebLogic Server Kubernetes Operator (the “operator”) which has several key features to assist you with deploying and managing WebLogic domains in a Kubernetes environment. You can:
-
-* Create WebLogic domains in a Kubernetes PersistentVolume. This PersistentVolume can reside in an NFS file system or other Kubernetes volume types.
-* Create a WebLogic domain in a container image.
-* Override certain aspects of the WebLogic domain configuration.
-* Define WebLogic domains as a Kubernetes resource (using a Kubernetes custom resource definition).
-* Start servers based on declarative startup parameters and desired states.
-* Manage WebLogic configured or dynamic clusters.
+The WebLogic Server Kubernetes Operator (the “operator”) supports running your WebLogic Server and Fusion Middleware Infrastructure domains on Kubernetes, an industry standard, cloud neutral deployment platform. It lets you encapsulate your entire WebLogic Server installation and layered applications into a portable set of cloud neutral images and simple resource description files. You can run them on any on-premises or public cloud that supports Kubernetes where you've deployed the operator.
```

Comment:
We should make the same set of changes to the top-level README.md, which is the content displayed when you visit the GitHub project page.

Reply:
Will do.